athenz/utils.go

```diff
@@ -157,15 +181,36 @@ func flattenRoleMembers(list []*zms.RoleMember) []interface{} {
 	for _, m := range list {
 		name := string(m.MemberName)
 		expiration := timestampToString(m.Expiration)
+		review := timestampToString(m.ReviewReminder)
 		member := map[string]interface{}{
 			"name":       name,
 			"expiration": expiration,
+			"review":     review,
 		}
 		roleMembers = append(roleMembers, member)
 	}
 	return roleMembers
 }
 
+func expandRoleSettings(settings map[string]interface{}) (int32, int32) {
+	return int32(settings["token_expiry_mins"].(int)), int32(settings["cert_expiry_mins"].(int))
+}
```

Comment:
please get rid of this method. I think direct access to the settings array is more readable